### ISSUE TITLE
Feature Request: Folders in App Drawer

### ISSUE THREAD

**Issue by CSM-Fol** [farmerbb/Taskbar#193]
I would like to request for a feature of making app folders inside the Taskbar, similar to what can be done on PhoenixOS and Linux Mint. 

I don't really like the Grid and List being the only options to show my apps in the app drawer. I need organization, not an untidy list of apps that gives me headaches in finding which app is it I want to open.

**Comment by StacktraceException** [comment]
Good idea, although tags are better. Most things don't fit into just one category, folder etc.

**Comment by CSM-Fol** [comment]
> Good idea, although tags are better. Most things don't fit into just one category, folder etc. 

I mean, you will just have to manually apps per folder, instead of automatic categorization.

**Comment by StacktraceException** [comment]
Why do you assume this? Can't tags be defined manually too just like folders(?).

**Comment by CSM-Fol** [comment]
> Why do you assume this? Can't tags be defined manually too just like folders(?).

I haven't really seen apps organized according to tags. Probably the closest I can think of this is Instagram grouping posts according to hashtags 🤣
